bftengine/include/bftengine/ReconfigurationCmd.hpp

```diff
@@ -106,6 +106,21 @@ class ReconfigurationCmd : public bftEngine::ResPagesClient<ReconfigurationCmd,
     metrics_.UpdateAggregator();
     return true;
   }
+  std::optional<uint64_t> getReconfigurationCommandEpochNumber() {
```

Comment:
This method will be only used by the read-only replica?

Reply:
yes